code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -4181,6 +4305,12 @@ Follow by example and make good judgement based on length which list to include
 			icon_state = "hand_panel_l"
 			body_parts = list(BP_L_HAND)
 
+	autopsy
```

Comment:
This is more of a personal opinion but I don't see this being logical to take ever. MAYBE think about leaving this one out.

Reply:
If other devs agree, I'll remove it.